packages/shared/ReactLazyComponent.js

```diff
@@ -38,11 +38,7 @@ export function initializeLazyComponentType(
   lazyComponent: LazyComponent<any>,
 ): void {
   if (lazyComponent._status === Uninitialized) {
-    let ctor = lazyComponent._result;
-    if (!ctor) {
-      // TODO: Remove this later. THis only exists in case you use an older "react" package.
-      ctor = ((lazyComponent: any)._ctor: typeof ctor);
-    }
+    const ctor = lazyComponent._result;
```

Comment:
Keep in mind that until we release this in isomorphic, it will break RN OSS.

Reply:
Makes sense. I won't do another full sync until after we've cut a stable version.